package/x86_64/RHEL_7/package.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -e
```

Comment:
Make this file executable. Also, how was this tested?

Reply:
Hey I verified by running the script on RHEL and it was creating the binary and then ran that binary to make sure its working.